src/auto/style_context.rs

```diff
@@ -170,6 +171,8 @@ pub trait StyleContextExt: 'static {
 
     fn get_path(&self) -> Option<WidgetPath>;
 
+    fn get_property(&self, property: &str, state: StateFlags) -> glib::Value;
```

Comment:
And also this should be `Option`

Reply:
This can probably be configured in the toml by marking the return type as nullable